### ISSUE TITLE
HbaseScheme assumes utf-8 encoded blob

### ISSUE THREAD

**Issue by kul** [Cascading/cascading.hbase#3]
Is there a particular reason for doing this ?
https://github.com/Cascading/cascading.hbase/blob/2.2/src/main/java/cascading/hbase/HBaseScheme.java#L141

This implies that i cannot have HBaseTap working on anything other than utf-8 encoded data. Is there a way to have raw bytes or a custom de/serialization for the tap?

**Comment by andrewnet** [comment]
First of all, the change referred in #3 are not in the latest branches. How this can be done?
Second, the change only affects 'source' method, how about 'sink' method?

The goal is to be able to 'read' and 'write' raw bytes. Since the raw bytes can be any thing, so they do not confirm to any encoding.

**Comment by fs111** [comment]
No, we have never changed this. Would you mind sending a patch to support that?
